tests/test_azure_sentinel.py

```diff
@@ -0,0 +1,22 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""Azure Sentinel unit tests."""
+from pytest import raises
+
+from msticpy.data.azure_sentinel import AzureSentinel
```

Comment:
We should try to create mocks for this and AzureData - feels like we should be able to get pretty good coverage (albeit with mocks and not the real API)

Reply:
I've done Azure Sentinel tests, I'll come back and increase Azure Data coverage at a later date.